api/src/AsyncRunner.php

```diff
@@ -64,17 +64,17 @@ public function cleanUp() {
 	}
 
 	/**
-	 * 
-	 * @throws AsyncRunnerException
+	 * Waits for up to 5s for the runner to complete
```

Comment:
```suggestion
	 * Waits for up to 50s for the runner to complete
```
50 seconds?

Reply:
No, it's really only 5s.

I went for a low value, because:
- Chorus times out after 30 seconds.
- This is only part of the request processing.
- The last chunk will generally land here twice (i.e. 10s)
- It doesn't matter if the number is too low, it just means the client will poll a bit more often

Reply:
ok sounds good.